src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/model/request/CreateVideoBookingRequest.kt

```diff
@@ -67,8 +66,8 @@ data class CreateVideoBookingRequest(
   val comments: String?,
 
   @field:Size(max = 120, message = "The video link should not exceed {max} characters")
-  @Schema(description = "The video link for the appointment. Must be a valid URL", example = "https://video.here.com")
-  val videoLinkUrl: String?,
+  @Schema(description = "The video link for the appointment.", example = "https://video.here.com")
+  val videoLinkUrl: String? = null,
```

Comment:
Now this is just free text perhaps want to handle whitespace, either trim it or don't allow it?